Need to divide the man page synopsis into two lines otherwise it's truncated.
Also remove a Linux specific comment.

--- httping-1.5.7/httping.1.orig	2013-02-15 07:48:07.796654766 -0800
+++ httping-1.5.7/httping.1	2013-02-15 07:49:39.972784905 -0800
@@ -11,7 +11,8 @@
 .BI "httping [" options "]
 .sp
 options:
-.BI "[\-g url] [\-h hostname] [\-p portnumber] [\-x proxyhost:port] [\-c count] [\-i interval] [\-t timeout] [\-s] [\-G] [\-b] [\-L xferlimit] [\-X] [\-l] [\-z] [\-f] [\-m] [\-o rc,...] [\-e string] [\-I useragent string] [\-R referer string] [\-r] [\-n warn,crit] [\-N mode] [\-q] [\-V]"
+.BI "[\-g url] [\-h hostname] [\-p portnumber] [\-x proxyhost:port] [\-c count] [\-i interval] [\-t timeout] [\-s] [\-G] [\-b] [\-L xferlimit] [\-X] [\-l] [\-z] [\-f] [\-m] [\-o rc,...] [\-e string]"
+.BI "[\-I useragent string] [\-R referer string] [\-r] [\-n warn,crit] [\-N mode] [\-q] [\-V]"
 .SH DESCRIPTION
 The program
 .B httping
@@ -104,7 +105,7 @@
 Activate the basic authentication, Username follow the -U, Password the -P.
 .TP
 .B "\-F"
-Attempt TCP Fast Open while trying to connect to a server (for Linux kernel 3.7.0 onwards)
+Attempt TCP Fast Open while trying to connect to a server.
 .TP
 .B "\-V"
 Show the version and exit.
